tasks/deploy.yaml

```diff
@@ -23,14 +23,14 @@ tasks:
       - description: "Deploy the Istio package, if UDS_PKG != istio"
         cmd: |
           if [ "${UDS_PKG}" != "istio" ]; then
-            uds zarf package deploy build/zarf-package-uds-core-istio-${UDS_ARCH}.tar.zst --confirm --no-progress
+            uds zarf package deploy build/zarf-package-uds-core-istio-${UDS_ARCH}.tar.zst --confirm --no-progress --components '*'
           fi
       - description: "Deploy the Pepr Module"
         cmd: |
           PEPR_VERSION=$(npm pkg get version | tr -d '"')
           uds zarf package deploy build/zarf-package-pepr-uds-core-${UDS_ARCH}-${PEPR_VERSION}.tar.zst --confirm --no-progress --set UDS_SINGLE_TEST=true
       - description: "Deploy the requested Zarf Package (must set UDS_PKG environment variable)"
-        cmd: uds zarf package deploy build/zarf-package-uds-core-${UDS_PKG}-${UDS_ARCH}.tar.zst --confirm --no-progress
+        cmd: uds zarf package deploy build/zarf-package-uds-core-${UDS_PKG}-${UDS_ARCH}.tar.zst --confirm --no-progress --components '*'
```

Comment:
Added this in so that the optional components get automatically deployed for single package tests.  This command is only used in testing single packages. I think it is safe to assume when run the validations/tests we want all components to be deployed.

Reply:
This makes a ton of sense, very clever approach to make sure we always test all components.